dwh/indicators.py

```diff
@@ -26,21 +26,26 @@
 import query_lib
 
 
-_CODE_SYSTEM = 'http://snomed.info/sct'
-# Fake codes for SNOMED-CT that matches our test data.
-_VL_CODE = '50373000'  # Height
-_ARV_PLAN = '106230009'  # Diagnosis certainty
-_DRUG1 = '410596003'  # Likely outcome
-_DRUG2 = '395098000'  # Disorder confirmed
-_TX_TB_PLAN = '106230009'  # Diagnosis certainty
-_TB_TEST_STATUS = '106230009'  # Diagnosis certainty
-# TODO: Generate representable dataset corresponding to snomed
-_YES_CODE = '410596003'  # dummy code for yes
-_CONTINUE_REGIMEN = '410596003'  # Likely outcome
-_START_DRUG = '410596003'  # Likely outcome
-_COMPLETE_REGIMEN = '410596003'  # Likely outcome
-_TB_PREV_plan = '106230009'  # Diagnosis certainty
-_TB_screening = '106230009'
+_CODE_SYSTEM='http://www.ampathkenya.org'
```

Comment:
@omarismail94 and @kimaina please take a deeper look at the code changes below. I have tried to make them based on the list in issue #179 and also by looking at the Synthea models and the codes in the generated dataset. But it is quite possible that these are wrong or incomplete. There is also one indicator below which I have commented out and need to investigate the synthetic data a little more.

Reply:
For the commented TX_TB indicator,  it is possible that we are not capturing scenarios where patients were screened for TB in  the semiannual reporting period who started TB treatment. Can we modify the synthe model to account for this scenario?

Reply:
What exactly would we be changing the model to?

 I've downloaded the `annual_df` dataframe as a CSV file (attached below), and would be interested to know what it should look like for the indicator to return non-empty

[annual.csv](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/files/7500638/annual.csv)

Reply:
Thank you @omarismail94  for downloading the dataframe, this is very useful. I now see there the problem is. @bashir2  can you try using `'6174' # REVIEW OF TUBERCULOSIS SCREENING QUESTIONS (6174)` instead of `2022' # TUBERCULOSIS DIAGNOSED THIS VISIT` as it doeasn't exist in the dataset. ?

Reply:
Thanks @omarismail94 and @kimaina for looking into this. I updated the code as Allan suggested and uncommented TX_TB.